docs/building-extensions/templates/template-details-file.md

```diff
@@ -17,7 +17,7 @@ The sections are defined as:
   <namespace><!-- Description of the template go here, usually CompanyNamespace\Templates\Templatename --></namespace>
 	<inheritable>1</inheritable>
 	<files><!-- Files/Folders for the template folder entries go here --></files>
-	<media destination="templates/site/cassiopeia" folder="media"><!-- Folders for the static assets entries go here --></media>
+	<media destination="templates/site/cassiopeia" folder="media"><!-- Folders for the static _assets entries go here --></media>
```

Comment:
Sorry it seems I miss communicated or you miss understood me, only in the manual we need to use the `_assets` folder not in joomla it self.

Reply:
nope that was clearly a mistake...